site-manage/pageorder/tool/src/bundle/org/sakaiproject/tool/pageorder/bundle/Messages.properties

```diff
@@ -48,6 +48,8 @@ error_pageid=A valid 'page ID' is required
 welcome=Changes to tool order will take effect upon 'Save'. When deleting or editing a tool name, changes will take effect immediately. Click and move tool in list below, or use the keyboard to focus on the tool then use U or D keys.
 warning=Warning: Making tools invisible does not prevent access to the tool items through direct links. To prevent all access, lock the tool.
 
+warn_lockResources=Locking the Resources tool may prevent other tools from functioning correctly, and is not recommended.
```

Comment:
Locking the Resources tool may prevent other tools from functioning correctly.

Reply:
```suggestion
warn_lockResources=Locking the Resources tool may prevent other tools from functioning correctly.
```